packages/fxa-auth-server/package.json

```diff
@@ -109,8 +109,9 @@
     "mysql": "^2.18.1",
     "node-zendesk": "^2.2.0",
     "nodemailer": "^6.9.9",
+    "openapi-fetch": "^0.13.5",
     "otplib": "^11.0.1",
-    "p-queue-compat": "^1.0.225",
+    "p-queue": "^8.1.0",
```

Comment:
With the version of node 22 we are using we can now `require(ESM)`.  The Recorded Future script will use `p-queue` but not in this PR (but I left the dep changes in).